app/src/main/java/org/schabi/newpipe/util/Localization.java

```diff
@@ -314,13 +318,33 @@ private static void initPrettyTime(final Context context) {
     }
 
     public static String relativeTime(final OffsetDateTime offsetDateTime) {
-        return relativeTime(GregorianCalendar.from(offsetDateTime.toZonedDateTime()));
+        return relativeTime(fromUTC(offsetDateTime));
```

Comment:
The tests are not covering the bug that you fixed. I can change back this line and your tests still pass successfully